README.md

```diff
@@ -97,50 +96,50 @@ Premade boilerplates for Fiber. **(WIP)**
 - [embedmode/fiberseed](https://github.com/embedmode/fiberseed)
 
 
-## Recipes
+## 📁 Recipes
 Recipes for Fiber. **(WIP)**
-- https://github.com/gofiber/recipes
+- [recipes](https://github.com/gofiber/recipes) - Official Fiber cookbook
```

Comment:
I think we don't need descriptions for recipes.

Reply:
actually wanted to add a description everywhere so that users know directly what they are looking at and it looks consistent everywhere